<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstructorWindow</class>
 <widget class="QMainWindow" name="ConstructorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конструктор изделия</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбранные детали</string>
    </property>
    <widget class="QScrollArea" name="SelectedScrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>331</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="SelectedScrollAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>389</width>
        <height>329</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Детали на складе</string>
    </property>
    <widget class="QScrollArea" name="StoredScrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>331</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="StoredScrollAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>389</width>
        <height>329</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="BuildButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Собрать изделие</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Код изделия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CodeInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameInput">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Наименование изделия</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StoreSelect">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Склад</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
